src/main/java/com/sigopt/model/Experiment.java

```diff
@@ -162,61 +166,6 @@ public PropertyResource(String prefix, String name, Class<T> klass) {
         }
     }
 
-    public static class Subresource<T extends APIObject> extends BoundObject {
```

Comment:
this might break client code that references this class, so this would need a major version bump

or you might be able to leave this and have it extend the other Subresource

Reply:
added it back here so still available in same scope